util/src/main/java/tc/oc/pgm/util/tablist/PlayerTabEntry.java

```diff
@@ -47,7 +48,9 @@ public PlayerTabEntry(Player player) {
   @Override
   public BaseComponent getContent(TabView view) {
     return TextTranslations.toBaseComponent(
-        PlayerComponent.of(player, NameStyle.TAB), view.getViewer());
+        PlayerComponent.of(player, NameStyle.TAB)
+            .decoration(TextDecoration.BOLD, view.getViewer().equals(player)),
```

Comment:
Should this be done inside the `PlayerComponent` renderer?

Reply:
Gotcha 👍 
I moved the rendering to the `PlayerComponent` though it did introduce a `viewer` parameter. Also I opted to not add back the `showSelf` to NameStyle, as the bolding of names will only be done in the tab. Does not make sense to define it if other styles will not utilize it. Let me know if this was the right approach or if you have an alternative.